test/tests/get-browser-info-test.js

```
@@ -0,0 +1,19 @@
+var expect         = require('chai').expect;
+var browserNatives = require('../../lib/index');
+
+describe('getBrowserInfo', function () {
+    it('Should raise an error if browser is not an alias or a path', function (done) {
```

Should raise an error if the browser ID is not an alias or file path